ros2doctor/ros2doctor/api/format.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def print_term(k, v):
+    """Print term only if it exists."""
+    if v:
+        # TODO: 20 padding needs to be dynamically set
```

Comment:
nit, add parenthesis with your github username. `TODO(claireyywang)`. The name in the parenthesis is the person who can answer questions about what needs to be done. Most of the time it's the person who created the comment.